recipes/smurff/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "0.17.0" %}
+
+package:
+  name: smurff
+  version: {{ version }}
+
+source:
+  url: https://github.com/ExaScience/smurff/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 998bb056b76c87feba4af2f1692236ffa420cc547f838a93262c8b846b8dea5d 
+
+build:
+    number: 0
+    script_env:
+      - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - mkl-devel
```

Comment:
It looks like `mkl` is not strictly necessary, but a blas/lapack library is. Please see conda-forge docs on how to use blas libraries.